scripts/v.dissolve/v.dissolve.html

```diff
@@ -7,6 +7,104 @@ <h2>DESCRIPTION</h2>
+or using SQL syntax in <b>aggregate_column</b>.
+If <b>result_column</b> is provided including type information
+and the <em>sql</em> backend is used,
+<b>aggregate_column</b> can contain SQL syntax specifying both columns
+and the functions applied, e.g.,
+<code>aggregate_column="sum(cows) / sum(animals)"</code>.
+In this case, <b>aggregate_methods</b> needs should be omitted.
+This provides the highest flexibility and it is suitable for scripting.
+
+<p>
+The backend is, by default, determined automatically based on the requested
+methods. Specifically, the <em>sql</em> backend is used by default,
+but when a method is not one of the SQLite build-in aggregate functions
+and, at the same time, is available with the <em>univar</em> backend,
+the <em>univar</em> backed is used.
+The default behavior is intended for interactive use and testing.
+For scripting and other automated usage, specifying the backend explicitly
+is strongly recommended.
+
+<p>
+For convince, certain methods, namely <em>n</em>, <em>count</em>,
+<em>mean</em>, and <em>avg</em>, are converted to the name appropriate
+for the selected backend. However, for scripting, specifying the appropriate
+method (function) name for the backend is recommended because the conversion
+is a heuristic which may change in the future.
+
+<p>
+If only <b>aggregate_column</b> is provided, methods default to
+<em>n</em>, <em>min</em>, <em>max</em>, <em>mean</em>, and <em>sum</em>.
+If the <em>univar</em> backend is specified, all the available methods
+for the <em>univar</em> backend are used.
+
+<p>
+If the <b>result_column</b> is not provided, each method is applied to each
+specified column producing result columns for all combinations. These result
+columns have auto-generated names based on the aggregate column and method.
+If the <b>result_column</b> is provided, each method is applied only once
+to the matching column in the aggregate column list and the result will be
+available under the name of the matching result column. In other words, number
+of items in <b>aggregate_column</b>, <b>aggregate_method</b> (unless omitted),
+and <b>result_column</b> needs to match and no
+combinations are created on the fly.
+For scripting, it is recommended to specify all resulting column names,
+while for interactive use, automatically created combinations are expected
+to be beneficial, especially for exploratory analysis.
+
+<p>
+Type of the result column is determined based on the method selected.
+For <em>n</em> and <em>count</em>, the type is INTEGER and for all other
+methods, it is DOUBLE. Aggregate methods which produce other types
+require the type to be specified as part of the <b>result_column</b>.
+A type can be provided in <b>result_column</b> using the SQL syntax
+<code>name type</code>, e.g., <code>sum_of_values double precision</code>.
+Type specification is mandatory when SQL syntax is used in
+<b>aggregate_column</b> (and <b>aggregate_method</b> is omitted).
+
 <h2>NOTES</h2>
```

Comment:
Add examples

Reply:
Plenty of new examples are now included.